candidate-standard/clause_7_core.adoc

```diff
@@ -147,20 +147,20 @@ An EDR resource is a collection of spatiotemporal data that can be sampled using
 
 EDR resources can be exposed using two path templates:
 
-* /collections/{collectionId}/{queryType}
-* /collections/{collectionId}/instances/{instanceId}/{queryType}
+* `/collections/{collectionId}/{queryType}`
+* `/collections/{collectionId}/instances/{instanceId}/{queryType}`
 
 Where 
 
-{collectionId} = a unique identifier for a collection of geospatial data.
+`{collectionId}` = a unique identifier for a collection of geospatial data.
 
-{instanceId} = a text string identifying the _'Version'_ or _'Instance'_ of the chosen collection.
+`{instanceId}` = a text string identifying the _'Version'_ or _'Instance'_ of the chosen collection.
 
-{queryType} = a text string identifying the query pattern performed by the API.
+`{queryType}` = a text string identifying the query pattern performed by the API.
 
 Information Resources associated with a specific collection should be accessed through the `/collections` path. Those which are not associated with a specific collection should use the `/{instanceId}/{queryType}` template.
 
-The EDR API `instanceId` parameter allows support for multiple iterations or versions of the same underlying datasource to be accessed by the API.  This is applicable when the entire datasource has been regenerated rather than individual values in the datasource being changed.  If only one instance of the datasource exists a value of _default_ or _latest_ could be used.
+The EDR API `instanceId` parameter allows support for multiple iterations or versions of the same underlying datasource to be accessed by the API.  This is applicable when the entire datasource has been regenerated rather than individual values in the datasource being changed.  If only one instance of the datasource exists a value of `_default_` or `_latest_` could be used.
```

Comment:
I think this entire section can now be removed,  as `instances` are now optional